repldex/config/__init__.py

```diff
@@ -25,7 +25,7 @@
 	EDITOR_IDS = [int(item.split()[0]) for item in f.read().splitlines()]
 	with open(config_path + 'entry_approval.txt') as f:
 		approves = f.read()
-		APPROVAL_IDS = [int(item.split()[0]) for item in approves.splitlines()]
+		APPROVAL_IDS = [item.split()[0] for item in approves.splitlines()]
```

Comment:
Bruah that int() is needed. Pls add back

Reply:
but it fails checks

Reply:
ohh wait nevermind

Reply:
Lmao